docs/apache-airflow-providers-amazon/executors/batch-executor.rst

```diff
@@ -0,0 +1,318 @@
+   container instances that are created for your compute environment to make calls to the
+   required AWS API operations on your behalf.
+
+3. Select a VPC which allows access to internet, as well as a security group with the necessary permissions.
+
+4. Click Next.
+
+Create a Job Queue
+~~~~~~~~~~~~~~~~~~
+
+1. Select a name for the job queue, as well as a priority. The compute environment will be set to the one created in the previous step.
+
+
+
+Create a Job Definition
+~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Choose a name for the Job Definition.
+
+2. Select the appropriate platform configurations. Ensure ``Assign public IP`` is enabled.
+
+3. Select an Execution Role, and ensure the role has the required permissions to accomplish its tasks.
+
+4. Enter the image URI of the image that was pushed in the previous section. Make sure the role being used has the required permissions to pull the image.
+
+5. Select an appropriate Job role, keeping in mind the requirements of the tasks being run.
+
+6. Configure the environment as required. You may specify the number of vCPU's, memory or GPUs available to the container.
+   Additionally, add the following environment variables to the container:
+
+- ``AIRFLOW__DATABASE__SQL_ALCHEMY_CONN``, with the value being the PostgreSQL connection string in the following format using the values set during the `Database section <#create-the-rds-db-instance>`_ above:
+
+.. code-block:: bash
+
+   postgresql+psycopg2://<username>:<password>@<endpoint>/<database_name>
+
+7. Add other configuration as necessary for Airflow generally (see `here <https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html>`__), the Batch executor (see :ref:`here <config-options>`) or for remote logging (see :ref:`here <logging>`). Note that any configuration changes should be made across the entire Airflow environment to keep configuration consistent.
+
+8. Click Next.
+
+9. In Review and Create page, review all the selections, and once everything is correct, click Create Resources.
+
+Allow Containers to Access RDS Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As a final step, access to the database must be configured for the containers managed by Batch. Many different networking configurations are possible, but one possible approach is:
+
+1. Log in to your AWS Management Console and navigate to the VPC Dashboard.
+
+2. On the left hand, under the Security heading, click Security groups.
+
+3. Select the security group associated with your RDS instance, and click Edit inbound rules.
+
+4. Add a new rule that allows PostgreSQL type traffic to the CIDR of the subnet(s) associated with the Batch Compute Environment.
+
+Configure Airflow
+~~~~~~~~~~~~~~~~~
+
+To configure Airflow to utilize the Batch Executor and leverage the resources we've set up, create a script (e.g., ``batch_executor_config.sh``) with the following contents:
```

Comment:
I wouldn't suggest the script thing, that's a pretty specific approach. The important thing to convey is that these Airflow configuration changes need to be made (in airflow.cfg, env vars, etc)